projects/1.21/assets/butchercraft/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,321 @@
+  "Butchercraft.jei.hook": "挂肉钩",
+  "block.butchercraft.barn_wood_block": "畜棚木板",
+  "block.butchercraft.barn_wood_door": "畜棚木门",
+  "block.butchercraft.barn_wood_fence": "畜棚木栅栏",
+  "block.butchercraft.barn_wood_fence_gate": "畜棚木栅栏门",
+  "block.butchercraft.barn_wood_slab": "畜棚木台阶",
+  "block.butchercraft.barn_wood_stairs": "畜棚木楼梯",
+  "block.butchercraft.barn_wood_trapdoor": "畜棚木活板门",
+  "block.butchercraft.beef_block": "生牛肉块",
+  "block.butchercraft.blood_fluid_block": "血液",
+  "block.butchercraft.blood_sausage_block": "生血肠块",
+  "block.butchercraft.butcher_block": "案板",
+  "block.butchercraft.chicken_block": "生鸡肉块",
+  "block.butchercraft.chicken_head": "鸡头",
+  "block.butchercraft.chicken_skull_head": "鸡头骨",
+  "block.butchercraft.cooked_beef_block": "熟牛肉块",
+  "block.butchercraft.cooked_blood_sausage_block": "熟血肠块",
+  "block.butchercraft.cooked_chicken_block": "熟鸡肉块",
+  "block.butchercraft.cooked_goat_block": "熟山羊肉块",
+  "block.butchercraft.cooked_mutton_block": "熟羊肉块",
+  "block.butchercraft.cooked_pork_block": "熟猪肉块",
+  "block.butchercraft.cooked_rabbit_block": "熟兔肉块",
+  "block.butchercraft.cooked_sausage_block": "熟香肠块",
+  "block.butchercraft.cow_head": "牛头",
+  "block.butchercraft.cow_hide_carpet": "牛皮",
+  "block.butchercraft.cow_skull_head": "牛头骨",
+  "block.butchercraft.goat_block": "生山羊肉块",
+  "block.butchercraft.goat_head": "山羊头",
+  "block.butchercraft.goat_hide_carpet": "山羊皮",
+  "block.butchercraft.goat_skull_head": "山羊头骨",
+  "block.butchercraft.grinder": "绞肉机",
+  "block.butchercraft.meat_hook": "挂肉钩",
+  "block.butchercraft.mutton_block": "生羊肉块",
+  "block.butchercraft.pig_head": "猪头",
+  "block.butchercraft.pig_hide_carpet": "猪皮",
+  "block.butchercraft.pig_skull_head": "猪头骨",
+  "block.butchercraft.pork_block": "生猪肉块",
+  "block.butchercraft.rabbit_black_head": "黑色兔头",
+  "block.butchercraft.rabbit_block": "兔肉块",
+  "block.butchercraft.rabbit_brown_head": "棕色兔头",
+  "block.butchercraft.rabbit_gold_head": "金色兔头",
+  "block.butchercraft.rabbit_salt_head": "胡椒盐色兔头",
+  "block.butchercraft.rabbit_skull_head": "兔头骨",
+  "block.butchercraft.rabbit_splotched_head": "斑点兔头",
+  "block.butchercraft.rabbit_white_head": "白兔头",
+  "block.butchercraft.sheep_head": "绵羊头",
+  "block.butchercraft.sheep_skull_head": "绵羊头骨",
+  "block.butchercraft.taxidermy_black_rabbit_block": "黑色兔子标本",
+  "block.butchercraft.taxidermy_brown_rabbit_block": "棕色兔子标本",
+  "block.butchercraft.taxidermy_chicken_block": "鸡标本",
+  "block.butchercraft.taxidermy_cow_block": "牛标本",
+  "block.butchercraft.taxidermy_goat_block": "山羊标本",
+  "block.butchercraft.taxidermy_gold_rabbit_block": "金色兔子标本",
+  "block.butchercraft.taxidermy_pig_block": "猪标本",
+  "block.butchercraft.taxidermy_salt_rabbit_block": "胡椒盐色兔子标本",
+  "block.butchercraft.taxidermy_sheep_block": "山羊标本",
+  "block.butchercraft.taxidermy_splotched_rabbit_block": "斑点兔子标本",
+  "block.butchercraft.taxidermy_white_rabbit_block": "白色兔子标本",
+  "block.butchercraft.trimmed_barn_wood_block": "染边畜棚木板",

```

```suggestion
  "block.butchercraft.trimmed_barn_wood_block": "纹边畜棚木板",
```
```suggestion
  "block.butchercraft.trimmed_barn_wood_block": "纹边畜棚木板（连接纹理）",
```
个人其实倾向第一个，但原文带着CTM的括注

---

已应用第一条建议